src/helpers/eventHelper.js

```diff
@@ -0,0 +1,90 @@
+import moment from 'moment';
+
+async function getEventsPresenters() {
+  var dateFilter = new Date().toISOString();
+  var oDataFilterOngoing = `$filter=Enabled ne false and EndDateTime ge datetime'${dateFilter}'%26$select=StartDateTime,Presenter%26$orderby=StartDateTime asc%26$top=50`;
+  var presenters;
+  await fetch(
+    `https://www.ssw.com.au/ssw/SharePointEventsService.aspx?odataFilter=${oDataFilterOngoing}`
+  )
+    .then(response => response.text())
+    .then(result => {
+      var parser = new DOMParser();
+      var xmlDoc = parser.parseFromString(result, 'application/xml');
+      presenters = xmlDoc.getElementsByTagName('Presenter');
+    });
+  return presenters;
+}
+
+async function getEventsForPresenter(name, nickname) {
+  var dateFilter = new Date().toISOString();
+  var oDataFilterOngoing = `$filter=(substringof('${name}',Presenter) or substringof('${nickname}',Presenter)) and Enabled ne false and EndDateTime ge datetime'${dateFilter}'%26$select=StartDateTime,EndDateTime,Category,CalendarType,Title,Url,Thumbnail,Presenter,EventShortDescription%26$orderby=StartDateTime asc%26$top=50`;
+  var events;
+  await fetch(
```

Comment:
Should you add a catch in case an error happens?

Reply:
Done, I return an empty list on error